code_to_gast/loop/js_loop.py

```diff
@@ -0,0 +1,17 @@
+import js_router as jr
+
+def while_statement_to_gast(node):
+    gast = {}
+    gast["type"] = "whileStatement"
+
+    """
+    Code re-used from js_conditional. TODO: write a helper or figure out how to go through router
+    """
+    body = node.body
+    if body.type == "BlockStatement":
+        gast["body"] = jr.node_to_gast(body)
+    else:
+        gast["body"] = [jr.node_to_gast(body)]
+
+    gast["test"] = jr.node_to_gast(node.test)
```

Comment:
Is test the condition of the while loop? If so, IMO condition is a clearer name.

Reply:
Both python and javascript use "test" which is why we used it again. It's also what is used in if statements.